app/helpers/toolbar_helper.rb

```diff
@@ -191,9 +191,11 @@ def toolbar_button_normal(props, view_button = false)
   # attributes in html
   #
   def data_hash_keys(props)
-    %i(pressed popup console_url prompt explorer confirm onwhen url_parms url).each_with_object({}) do |key, h|
+    hsh = %i(pressed popup console_url prompt explorer onwhen url_parms url).each_with_object({}) do |key, h|
       h["data-#{key}"] = props[key] if props.key?(key)
     end
+    hsh["data-confirm"] = _("#{props[:confirm]}") if props.key?(:confirm)
+    hsh
```

Comment:
Can you, please, move this bit to the `prepare_tag_keys` method so that we sort of stick with the logic and naming of the methods? (same as you do with `title`).

Thx!